packages/react-liveness/src/components/FaceLivenessDetector/service/types/machine.ts

```diff
@@ -1,21 +1,50 @@
 import { ActorRef, Interpreter, State } from 'xstate';
 import {
-  ValidationException,
   InternalServerException,
-  ThrottlingException,
+  ChallengeConfig,
+  ColorSequence,
+  LightChallengeType,
+  OvalParameters,
   ServiceQuotaExceededException,
-  SessionInformation,
+  SessionInformation as ServerSessionInformation,
+  ValidationException,
+  ThrottlingException,
 } from '@aws-sdk/client-rekognitionstreaming';
-
+import {
+  FACE_MOVEMENT_CHALLENGE,
+  FACE_MOVEMENT_AND_LIGHT_CHALLENGE,
+} from '../utils';
+import { ErrorState } from './error';
+import { Face, FaceDetection } from './faceDetection';
 import {
   FaceLivenessDetectorCoreProps,
   FaceMatchState,
   LivenessOvalDetails,
   IlluminationState,
 } from './liveness';
-import { ErrorState } from './error';
 import { StreamRecorder, ColorSequenceDisplay } from '../utils';
-import { Face, FaceDetection } from './faceDetection';
+
+interface Challenge {
+  name: string;
+}
+
+export interface FaceMovementAndLightChallenge extends Challenge {
+  name: (typeof FACE_MOVEMENT_AND_LIGHT_CHALLENGE)['Type'];
+  ChallengeConfig: ChallengeConfig | undefined;
+  ColorSequences: ColorSequence[] | undefined;
+  LightChallengeType: LightChallengeType | undefined;
+  OvalParameters: OvalParameters | undefined;
+}
+
+export interface FaceMovementChallenge extends Challenge {
+  name: (typeof FACE_MOVEMENT_CHALLENGE)['Type'];
+  ChallengeConfig: ChallengeConfig | undefined;
+  OvalParameters: OvalParameters | undefined;
+}
```

Comment:
That makes sense. My rationale was that we basically want to copy their types exactly (w the addition of name), but don't think there's any real benefit so down to make them lowercase.

Reply:
Discussed offline and realized it may make more sense to directly extend from rekognition's challenge types and then rename `Name` to match the casing but curious if anyone has opinions about this.